### ISSUE TITLE
Macro to convert back from lootable

### ISSUE THREAD

**Issue by UndeadSwordsman** [DFreds/dfreds-pocket-change#22]
If this is a feature, I'm unable to figure it out, sorry. I don't understand macros very well, but I didn't see anything in the ones you've provided that might do this.

Once the party has looted the creature, it would be nice if there was a way to convert it back to normal. Return it to NPC sheet, remove permissions and remove treasure chest overlay (Particularly the overlay, as I can do the other stuff manually if I need to).

-One of the reasons this would be helpful is that sometimes the party would leave a corpse and come back to it later, like a body in a crime scene. The overlay lowers immersion in that particular scene. 
-I also like to keep track of what was looted, in case issues arise later where items disappear from bugs or an accidental wipe of inventory from DNDBeyond Importer; I keep bodies on the scene, but hide them after (I keep what was on corpses in a seperate note on the npc). A quicker way to remove permissions would be helpful, so I can mass hide them.
-Or, sometimes the corpses get in the way of combat, with them tiling over each other and players having a hard time selecting themselves, so I hide them but leave them in the same spot, to unhide them after they've moved from the spot. Hiding them, while the players have permissions to observe, leaves it visible to the players. Returning it to a regular NPC, with whatever items or gold is still on the body, then hiding it with a quick right-click, makes it take less time in combat. I can then quickly return it to a lootable item later.

These are just my own reasons, and it's not necessary. It would help smooth things out while playing, keeping everything in the flow, while I manage my players and the canvas.

Thank you for taking the time to read this.

**Comment by DFreds** [comment]
That's worth considering. Just note that once it is converted to loot and removes all the features or does the damaged items or whatever, it can't be converted back in exactly the same way. Otherwise, I'll look into it.

**Comment by CaosFR** [comment]
This macro revert to NPC but like DFreeds, we loose must of thing :

```
async function main() {
  for (let token of canvas.tokens.controlled) {
    await token.document.actor.setFlag('core', 'sheetClass', 'Default');
    await token.document.update({
      'actorData.permission.default': CONST.ENTITY_PERMISSIONS.NONE,
      overlayEffect: '',
    });
  }
}
main();
```

**Comment by DFreds** [comment]
Released an official version of this in v3.1.6
